vyper/functions.py

```diff
@@ -156,15 +156,12 @@ def floor(expr, args, kwargs, context):
     )
 
 
-@signature(('num', 'decimal'))
```

Comment:
What was behind this change?

Reply:
There is no reason to do decimal(decimal), especially because we are statically typing now.